fxmanifest.lua

```diff
@@ -10,8 +10,13 @@ this_is_a_map 'yes'
 
 version '1.0.0'
 
-server_scripts {
+shared_scripts {
 	'config.lua',
+	'@qb-core/shared/locale.lua',
+	'locales/de.lua'
```

Comment:
Please, English as default language.

Reply:
Oh, yeah, I'll change that. Wait for me to come home.